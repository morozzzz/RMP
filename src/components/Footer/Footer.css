.footer {
    height: 3em;
    background-color: rgb(42, 42, 42);
    display: flex;
    align-items: center;
    justify-content: center;
}

.footer__content {
    width: 90%;
}