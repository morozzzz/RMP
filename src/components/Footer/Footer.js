import React from 'react';
import Logo from '../Logo/Logo';
import './Footer.css';

const Footer = () => (
    <div className="footer">
        <div className="footer__content">
            <Logo />
        </div>
    </div>
);

export default Footer;
