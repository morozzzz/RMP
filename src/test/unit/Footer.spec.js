import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../components/Footer/Footer';

jest.mock('../../components/PosterTile/PosterTile', () => () => <div id="PosterTile" />);

describe('Footer', () => {
    it('should be rendered correctly', () => {
        const component = shallow(<Footer />);

        expect(component).toMatchSnapshot();
    });
});
