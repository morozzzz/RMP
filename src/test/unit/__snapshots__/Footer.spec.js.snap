// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should be rendered correctly 1`] = `
<div
  className="footer"
>
  <div
    className="footer__content"
  >
    <Logo
      name="netflixroulettee"
      redirectUrl=""
    />
  </div>
</div>
`;
